packages/e2e-tests/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM node:20.11
+
+ARG uid
+ARG gid
+
+RUN apt update && apt upgrade -y
+RUN groupadd -g $gid -o noone
+RUN useradd -ms /bin/bash -u $uid -g $gid noone
+
+RUN npm install -g pnpm
+RUN pnpm config set store-dir /usr/local/pnpm-store
+
+# Copy the package.json to install playwright
+# /app is overriden by the volume mount in docker-compose.yml
+# to the local WORKDIR
+COPY ./packages/e2e-tests/package.json /app/package.json
```

Comment:
And doesn't this replace the root `package.json` with the `packages/e2e-tests/package.json` on the host machine?